generators/azure-app-service/index.js

```diff
@@ -454,8 +451,17 @@ which is free for the first 30 days`);
         this.log(`The Application Insights instrumentation key used is: '${chalk.bold(this.azureAppInsightsInstrumentationKey)}'`);
         done();
       },
+    };
+  }
+
+  loadPlatformConfig(config = _.defaults({}, this.jhipsterConfig, defaultConfig, this), dest = this) {
```

Comment:
```suggestion
  _loadPlatformConfig(config = _.defaults({}, this.jhipsterConfig, defaultConfig, this), dest = this) {
```
api method must be prefixed with `_`